add system variable back for tidb_enable_point_get_plan_cache. (#20679)
pingcap/docs: system-variables.md

</CustomContent>

### tidb_enable_point_get_cache

- Scope: SESSION
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): YES
- Type: Boolean
- Default value: `OFF`

> - When the table lock type of [`LOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) is set to `READ`, setting this variable to `ON` enables caching of point query results, reducing the overhead of repeated queries and improving point query performance.

Reviewer: ```suggestion
- When you set the table lock type of [`LOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) to `READ`, setting this variable to `ON` enables caching of point query results, reducing the overhead of repeated queries and improving point query performance.
```

- Possible values: `IN`
- Controls the type of predicate used by the generated Filter operator. Currently, only one type is supported: `IN`. For more information, see [Runtime Filter type](/runtime-filter.md#runtime-filter-type).

### tidb_enable_point_get_cache

- Scope: SESSION
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): YES
- Type: Boolean
- Default value: `OFF`

> - This variable can benefit the performance when table lock is enabled for type of "READ" and "READ ONLY" with this variable set to `ON`.

Reviewer: ```suggestion
- When the table lock type is set to `READ` or `READ ONLY`, enabling this variable can improve the performance of point queries.
```

> - When the table lock type is set to `READ` or `READ ONLY`, setting this variable to `ON` can enhance the performance of point queries by enabling point get plan cache.

Reviewer: ```suggestion
- When the table lock type is set to `READ` or `READ ONLY`, setting this variable to `ON` enables caching of point query results, reducing the overhead of repeated queries and improving point query performance.
```

> - When the table lock type is set to `READ` or `READ ONLY`, setting this variable to `ON` enables caching of point query results, reducing the overhead of repeated queries and improving point query performance.

Reviewer: ```suggestion
- When the table lock type of [`LOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) is set to `READ` or `READ ONLY`, setting this variable to `ON` enables caching of point query results, reducing the overhead of repeated queries and improving point query performance.
```
